swarm/storage/netstore.go

```diff
@@ -121,6 +121,11 @@ func (self *NetStore) Get(key Key) (chunk *Chunk, err error) {
 	}
 }
 
+// GetWithTimeout makes a single retrieval attempt for a chunk with a explicit timeout parameter
```

Comment:
I wish you could do away without this change. we sort all this once the chunk refactor is done. 
but ok.

Reply:
Let's call it an extra motivation to get the refactor right ;)